src/iOS/Info.plist

```diff
@@ -125,7 +125,5 @@
 	<string>Use Face ID to unlock your vault.</string>
 	<key>NFCReaderUsageDescription</key>
 	<string>Use Yubikeys for two-facor authentication.</string>
-	<key>UILaunchImages</key>
```

Comment:
Ah, the good ol' `UILaunchImages` key, I'm going to miss you old friend...... not, lol